src/index.js

```diff
@@ -1,54 +1,107 @@
 'use strict';
 
-const $tagsInput = document.getElementById('tags-input');
-const $tagsList = document.getElementById('tags-list');
-const tagsArr = ['1', 'super tag'];
+const ERROR_ClASS_NAME = 'invalid-value';
 
-$tagsInput.addEventListener('keyup', ({key, target}) => {
-  if (key === 'Enter' && target.value.trim()) {
-    const $el = createTagElement(target.value);
-    const isElementExist = tagsArr.includes(target.value);
+const createTagElement = content => {
+  const $li = document.createElement('li');
+  
+  $li.className = 'tags-list-item';
+  $li.innerHTML = `
+    <span class="tag-name">${content}</span>
+    <span class="remove-btn">x</span>
+  `;
+  
+  return $li;
+};
 
-    if (!isElementExist) {
+const createInputBlock = (inputTitle, countNumber) => {
+  const inputBlock = document.createElement('div');
+  
+  inputBlock.className = 'input-block';
+  inputBlock.innerHTML = `
+    <h2 class="title-level-2">${inputTitle}:</h2>
+    <input
+      id=${`input-${countNumber}`}
+      class="tags-input "
+      type="text"
+      placeholder="type your tags..."
+    />
+    <ul id=${`list-${countNumber}`} class="tags-list"></ul>
+  `;
+  
+  return inputBlock;
+};
+
+const initDefaultTags = (tags, $tagsList) => tags.forEach(tagValue => {
+  const $el = createTagElement(tagValue);
+  
+  $tagsList.appendChild($el);
+});
+
+const validateInputValue = ($input, isValueUniq) => {
```

Comment:
very odd construction. Try not use "return" without value

Reply:
got it.